### PR TITLE
DP-64: Add meta descriptions to each page in the operation guide.

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/DP-64

### CODE REVIEW

**Review by mmahacek — COMMENTED** [review]
just some minor questions

File: docs/modules/operation/pages/deep-dive/provisioning/detectors.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [[ga-detectors]]
 = Detectors
+:description: Overview of detectors in OpenNMS Horizon/Meridian to detect available services on nodes automatically. Intro. to metadata domain-specific language.
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/deep-dive/provisioning/detectors.adoc:3]: Hmm. I'm not sure if separate lines would affect the meta tag. I'll do some research.

[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/deep-dive/provisioning/detectors.adoc:3]: @mmahacek , according to Colby, the descriptions have to be one block of text. We can't do separate lines. As long as it's under 155 characters or so, it's ok, but no line breaks.

[the remark this answers is not in the document]
Comment (mmahacek) [reply docs/modules/operation/pages/deep-dive/provisioning/detectors.adoc]: Okay. I wasn't sure how these rendered compared to paragraph spacing in the body content.

**Review by mmahacek — APPROVED** [review]
I trust your words.

**Comment by Bonrob2** [comment]
> I trust your words.

Most of it is taken from the intros to the pages ... just shortened to fit the character limit, and to try to make things a bit catchier for people scanning. Thanks!

**Pull request merged**